person.rb

```diff
@@ -23,5 +26,37 @@ def can_use_services?
     end
   end
 
+  def correct_name
+    @name
+  end
+
   private :is_of_age?
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- Hey :wave: , let's kindly add the attr_accessor for the nameable so we can easily access it 
` attr_accessor :nameable`

Reply:
### TEST TO SEE THE DECORATION IN ACTION
###### Try the following code and check if you managed to decorate your person:



- Hey, please, kindly ensure you've added a file that contains this codes to test your decorator to see if it works correctly

```ruby
person = Person.new(22, 'maximilianus')
  person.correct_name
  capitalizedPerson = CapitalizeDecorator.new(person)
  capitalizedPerson.correct_name
  capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
  capitalizedTrimmedPerson.correct_name
```